DoubleSharp/Linq/LinqExtensions.cs

```diff
@@ -343,4 +344,41 @@ public static class LinqExtensions {
 	[Pure]
 	public static IEnumerable<T> Times<T>(this int count, Func<int, T> functor) =>
 		count.Range().Select(functor);
+
+    /// <summary>
+    /// Gets an array of <see cref="Type"/> objects representing the types of the tuple components.
+    /// </summary>
+    /// <param name="tuple">The tuple.</param>
+    /// <returns>An array of <see cref="Type"/> objects representing the types of the tuple components.</returns>
+    public static Type[] GetComponentTypes(this ITuple tuple)
+    {
+        return tuple.GetType().GetGenericArguments();
```

Comment:
I'm gonna write a little style guide, but we should favor expression bodies when possible, and have opening braces on the definition line. Not a blocker, just a note!